src/lib/index.js

```diff
@@ -42,4 +43,5 @@ export {
   Export,
   Router,
   Callback,
+  Login
```

Comment:
Add a hanging comma pls

Reply:
> Add a hanging comma pls

@ciremusyoka  I have my settings set on vscode to format on save. I find this easier

Reply:
@kelvin-muchiri what's the setting?

Reply:
> @kelvin-muchiri what's the setting?

```
{
  "printWidth": 100,
  "singleQuote": true,
  "trailingComma": "es5",
  "arrowParens": "avoid"
}
```

Reply:
> I noted that you install `js-cookie` but did not use it. Could you give me context about that?

Yea noticed that too. I haven't used it anywhere will remove that.